createdisk.sh

```diff
@@ -247,6 +247,9 @@ ${OC} --config $1/auth/kubeconfig create clusterrolebinding developer --clusterr
 certImage=$(${OC} --config $1/auth/kubeconfig adm release info --image-for=cluster-kube-apiserver-operator)
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo podman tag $certImage openshift/cert-recovery
 
+# Remove unused images from container stroage
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo crictl images -q | xargs -n 1 sudo crictl rmi 2>/dev/null'
```

Comment:
I would run this after the `systemctl stop kubelet` to avoid the (probably only theoritical for us) issue mentioned here https://github.com/kubernetes-sigs/cri-tools/issues/399#issuecomment-435562365
It probably would not hurt to explain in the commit log or in a comment that this is attempting to remove *all* images, but that crictl will not fail to remove images which are being used.

Reply:
@cfergeau If we run it after stopping the kubelet then it will remove the kubelet image also since as workaround it is just removing the images which are not used by any containers. Also `kubelet would pull an image before creating the container. If the prune happens during them, the container creation may be failed.` it mention that corner case happen if we are pulling the image during pruning the images and our case that is not true.

Reply:
> If we run it after stopping the kubelet then it will remove the kubelet image 

oh ok, we indeed don't want that :)

> it mention that corner case happen if we are pulling the image during pruning the images and our case that is not true.

we are not doing it, but some operator could theoritically decide to do that in the background for whatever reason. I agree this is most likely not going to happen, and since we cannot easily avoid it, this should be good enough for now.